Rakefile

```diff
@@ -0,0 +1,11 @@
+require 'bundler/gem_tasks'
+
+require 'rspec/core/rake_task'
+RSpec::Core::RakeTask.new
+
+# Not using RuboCop::RakeTask.new(:rubocop) since it's showing different errors than just running rubocop
```

Comment:
The default was to use this `RuboCop::RakeTask.new(:rubocop)` to create the rake task, but that was showing a different output than just running `rubocop` or `bundle exec rubocop`. I checked and the version was the same in both cases, so I didn't know why the output was different... When I run it with this task, I got the following errors:
```
lib/fastlane/plugin/revenuecat/actions/bump_version_update_changelog_create_pr_action.rb:56:11: C: Require/MissingRequireStatement: FastlaneCore::ConfigItem not found, you're probably missing a require statement or there is a cycle in your dependencies.
          FastlaneCore::ConfigItem.new(key: :current_version,
          ^^^^^^^^^^^^^^^^^^^^^^^^
lib/fastlane/plugin/revenuecat/actions/replace_version_number_action.rb:38:11: C: Require/MissingRequireStatement: FastlaneCore::ConfigItem not found, you're probably missing a require statement or there is a cycle in your dependencies.
          FastlaneCore::ConfigItem.new(key: :current_version,
          ^^^^^^^^^^^^^^^^^^^^^^^^
lib/fastlane/plugin/revenuecat/helper/revenuecat_helper.rb:15:9: C: Require/MissingRequireStatement: Action not found, you're probably missing a require statement or there is a cycle in your dependencies.
        Action.sh("sed", '-i', backup_extension, sed_regex, path)
        ^^^^^^
```
But when running rubocop directly, it didn't show any errors. Also, tests run fine, so I decided to just run `rubocop` directly instead of using this precreated rake task.

Reply:
Weird that it didn't show the same errors! Not sure why that would happen 🤷‍♂️ This works so that is what matters 😊